framework/src/utils/MooseMeshUtils.C

```diff
@@ -12,6 +12,7 @@
 
 #include "libmesh/elem.h"
 #include "libmesh/boundary_info.h"
+#include "libmesh/replicated_mesh.h"
```

Comment:
Ah - my apologies - I was looking at the commits here one at a time and didn't realize that this diff was restoring something that a previous commit had removed.

I do think it's not needed; looking over the code suggests that some idiot introduced it for no good reason ... but don't delete it, leave it in.  `git log` reveals that the idiot was *me*, and the log message gives a plausible reason; just in case I was correct and the reason is still subtly valid let's not take the risk of regressing something here.